windows/devices/uwb/include/windows/devices/uwb/UwbAppConfiguration.hxx

```diff
@@ -7,26 +7,29 @@
 #include <cstring>
 #include <memory>
 #include <tuple>
+#include <vector>
 
 #include <UwbCxLrpDeviceGlue.h>
 
 namespace windows::devices
 {
+class UwbSetAppConfigurationParametersBuilder;
 /**
- * @brief Adaptor for using UWB_APP_CONFIG_PARAM UwbCx DDI structure in a nicer way. 
- * 
+ * @brief Adaptor for using UWB_APP_CONFIG_PARAM UwbCx DDI structure in a nicer way.
+ *
  * @tparam PropertyT The type of property the UWB_APP_CONFIG_PARAM structure
  * holds as its flexible array member.
  */
 template <typename PropertyT>
 class UwbAppConfigurationParameter
 {
+    friend class UwbSetAppConfigurationParametersBuilder; // friend class so the unique buffer ptr can be moved out
```

Comment:
See my later comment about not `move`'ing this out. If you agree with that, I think this can be removed.